sympy/core/basic.py

```diff
@@ -837,6 +837,12 @@ def subs(self, *args, **kwargs):
         else:
             raise ValueError("subs accepts either 1 or 2 arguments")
 
+        if isinstance(self, sympy.core.symbol.Symbol) and isinstance(args[0], sympy.physics.vector.vector.Vector):
```

Comment:
@moorepants @debugger22 
I added a check in the `subs` function. But I get `global name sympy not defined` when I test it. How do I resolve this ?